### PR TITLE
feat: Add Examples to 'Examples' page

### LINKED ISSUE #4: create examples
## Description

For this you can test run the site and create example cover letters record it and create an example page where you'll add this video so users can see it and understand how the site works.

## Checklist

- [x] I have searched for similar issues before opening this one.
- [x] I have read and understand the [CONTRIBUTING.md](./CONTRIBUTING.md) document.

### PR DESCRIPTION
**Pull request by GrandeMan**
-Create Example component and routes to App.jsx

-Create demo vid and embed in Examples.jsx

-Add generated cover letters to Examples.jsx

-Add breakpoints and adhere to style of other pages

-Build and test

## Description

Create demo vid and cover letter examples 

## Checklist

- [x] I have read the [CONTRIBUTING.md](./CONTRIBUTING.md) document.
- [x] My code follows the coding guidelines of the project.
- [x] I have tested my changes thoroughly.
- [ ] I have added/updated relevant documentation.
- [ ] I have updated any necessary unit tests.
- [x] My commits follow the commit guidelines.

Fixes #4

### CODE REVIEW

File: client/src/App.jsx
Comment (Judge-Paul) [inline client/src/App.jsx:1]: Looks good, but your tabs configuration is affecting the App,jsx file and adding more spaces than it should.

If you want to use those spaces in the page you created that's fine but please leave the App.jsx to default formatting

**Comment by Judge-Paul** [comment]
@GrandeMan Please check the review and make the necessary changes so I can merge

Everything else looks good and thanks for your contribution.

**Review by GrandeMan — COMMENTED** [review]
Changes look good.

**Review by GrandeMan — COMMENTED** [review]
Good to go

**Comment by Judge-Paul** [comment]
You haven't made any new commits yet

**Comment by GrandeMan** [comment]
Sorry about that, I was having a problem with my prettier config. Should've just edited the code here (Github) from the beginning. The files should be good now

**Review by Judge-Paul — APPROVED** [review]
Everything looks good

**Comment by Judge-Paul** [comment]
@GrandeMan I saw the video when I was reviewing it but I didn't play it cause I thought it was just a no-audio recording of you using the site, but it's really impressive, sorry I didn't notice it sooner

Thanks a lot 🙏🏾

**Pull request merged**